core/src/main/java/edu/wpi/grip/core/http/PostHandler.java

```diff
@@ -0,0 +1,18 @@
+
+package edu.wpi.grip.core.http;
+
+/**
+ * Handler for converting incoming data over an HTTP {@code POST} operation.
+ */
+public interface PostHandler {
```

Comment:
You could make this an `@FunctionalInterface` and use it like a lambda instead of having classes implement it.